contracts/src/v0.8/functions/dev/1_0_0/FunctionsClient.sol

```diff
@@ -57,5 +57,6 @@ abstract contract FunctionsClient is IFunctionsClient {
       revert OnlyRouterCanFulfill();
     }
     fulfillRequest(requestId, response, err);
+    emit RequestFulfilled(requestId);
```

Comment:
nit: I wonder if it makes sense to also emit the response & error?

Reply:
It'll cost a teeny bit more gas, would this be valuable to users?
I don't have a strong opinon. If you do I can add, or we could run it by Dev Rel

Reply:
That's fine. Not worth holding this PR.